dna-frontend/src/app/core/components/input-slider/input-slider.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, Input } from '@angular/core';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { TuiInputSliderModule } from '@taiga-ui/kit';
+
+@Component({
+  selector: 'app-input-slider',
+  standalone: true,
+  imports: [FormsModule, ReactiveFormsModule, TuiInputSliderModule],
+  templateUrl: './input-slider.component.html',
+  styleUrl: './input-slider.component.scss',
+})
+export class InputSliderComponent {
```

Comment:
[SliderKeySteps]: You cannot programmatically set value which is less/more than min/max